rites/roman1969/__tests__/calendar-builder.test.ts

```diff
@@ -1,13 +1,11 @@
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
-import { England_En } from 'romcal/dist/bundles/england';
-import { GeneralRoman_En } from 'romcal/dist/bundles/general-roman';
-import { Germany_En } from 'romcal/dist/bundles/germany';
-import { Hungary_En } from 'romcal/dist/bundles/hungary';
-import { Ireland_En } from 'romcal/dist/bundles/ireland';
-import { Slovakia_Sk } from 'romcal/dist/bundles/slovakia';
-
-import { CalendarScope, Id, LiturgicalCalendar, LiturgicalDay, LiturgicalDayDef, Period, Romcal, Season } from '../lib';
-import { dateDifference } from '../lib/utils/dates';
+import { England_En } from '../dist/bundles/england';
+import { GeneralRoman_En } from '../dist/bundles/general-roman';
+import { Germany_En } from '../dist/bundles/germany';
+import { Hungary_En } from '../dist/bundles/hungary';
+import { Ireland_En } from '../dist/bundles/ireland';
+import { Slovakia_Sk } from '../dist/bundles/slovakia';
```

Comment:
if we're going to reference these bundle paths so much, i think we should wire up these paths in the tsconfig and resolve them as `@bundles/{calendar}` so that we can avoid quite so many messy paths.